### PR TITLE
Persistence Language Functionality to Store Language in User Preferences

### PR DESCRIPTION
**Pull request by oruburos**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named

### CODE REVIEW

File: client/modules/User/actions.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ export function validateAndLoginUser(previousPath, formProps, dispatch) {
           type: ActionTypes.SET_PREFERENCES,
           preferences: response.data.preferences
         });
+        const valorLanguage = response.data.preferences.language;
```
Comment (andrewn) [inline client/modules/User/actions.js:62]: What does `valorLanguage` mean?

Comment (oruburos) [reply client/modules/User/actions.js:62]: Hi, it is response.data.preferences.language , I forgot to use the same way to access it in both parts

Comment (andrewn) [reply client/modules/User/actions.js:62]: But I meant, what does the word `valor` mean as a variable name?

Comment (oruburos) [reply client/modules/User/actions.js:63]: it means value, so a proper name for this intermediate variable would be languageValue, not this thing, I will get rid of this.

Comment (oruburos) [reply client/modules/User/actions.js]: @andrewn I've deleted the variable.

**Comment by oruburos** [comment]
@andrewn I deleted the variable and merged a PR. Can you take a look, please?

**Comment by oruburos** [comment]
@andrewn I have merged the develop branched,  Can you have a look, please?

**Pull request merged**